cmd/in-toto/run.go

```diff
@@ -125,31 +127,40 @@ in environment variables or config files. See Config docs for details.`,
 }
 
 func runPreRun(cmd *cobra.Command, args []string) error {
-	key = intoto.Key{}
-	cert = intoto.Key{}
+	if spiffeUDS != "" {
+		ctx := context.Background()
+		var err error
+		key, err = spiffe.GetSVID(ctx, spiffeUDS)
+		if err != nil {
+			return fmt.Errorf("failed to get spiffe x.509 SVID: %w", err)
+		}
```

Comment:
Can we make runPreRun and recordPreRun the same function, or call a common "getKey" function between the two?

Reply:
Combined function.